### PR TITLE
SshSteps.readLog() public access

### PR DESCRIPTION
**Pull request by xsmrcek**
I would like to make readLog() public. So projects that are using some custom ssh logic, can utilize sshSteps too.

### CODE REVIEW

**Comment by paveljandejsek** [comment]
Generated [reports](http://95.168.205.146:8080/reports/187/index.xml)

**Comment by paveljandejsek** [comment]
Nice commit message :D

File: jbehave-support-core/src/main/java/org/jbehavesupport/core/ssh/SshSteps.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ public void init() {
         logCache.clear();
     }
 
-    private String readLog(String systemQualifier, ZonedDateTime startTime, ZonedDateTime endTime) {
+    public String readLog(String systemQualifier, ZonedDateTime startTime, ZonedDateTime endTime) {
```
Comment (mbocek) [inline jbehave-support-core/src/main/java/org/jbehavesupport/core/ssh/SshSteps.java]: I'm still not sure that this API should be public. We should discuss it again and probably vote.

Comment (xsmrcek) [reply jbehave-support-core/src/main/java/org/jbehavesupport/core/ssh/SshSteps.java]: This Is a reason for separate PR :)

**Pull request closed without merging**